public/controllers/management/components/management/mg-logs/logs.js

```diff
@@ -11,536 +11,575 @@
   EuiFlexGroup,
   EuiFlexItem,
-  EuiSelect,
-  EuiSwitch,
-  EuiSpacer,
   EuiPage,
   EuiPageBody,
-  EuiCodeBlock,
   EuiPanel,
-  EuiTitle,
-  EuiTextColor,
   EuiProgress,
-  EuiCallOut,
-  EuiButtonEmpty
+  EuiSelect,
+  EuiSpacer,
+  EuiSwitch,
+  EuiTextColor,
+  EuiTitle,
 } from '@elastic/eui';
-import 'brace/mode/less';	
-import 'brace/theme/github';	
+import 'brace/mode/less';
+import 'brace/theme/github';
 import exportCsv from '../../../../../react-services/wz-csv';
-import { getToasts }  from '../../../../../kibana-services';
+import { getToasts } from '../../../../../kibana-services';
 import { WzRequest } from '../../../../../react-services/wz-request';
 import { formatUIDate } from '../../../../../react-services/time-service';
-import { withUserAuthorizationPrompt, withGlobalBreadcrumb } from '../../../../../components/common/hocs';
+import {
+  withGlobalBreadcrumb,
+  withUserAuthorizationPrompt,
+} from '../../../../../components/common/hocs';
 import { compose } from 'redux';
 import { WzFieldSearch } from '../../../../../components/wz-field-search-bar/wz-field-search-bar';
+import { UI_LOGGER_LEVELS } from '../../../../../../common/constants';
+import { UI_ERROR_SEVERITIES } from '../../../../../react-services/error-orchestrator/types';
+import { ErrorOrchestratorService } from '../../../../../react-services';
 
 export default compose(
   withGlobalBreadcrumb([
     { text: '' },
     { text: 'Management', href: '/app/wazuh#/manager' },
-    { text: 'Logs' }
+    { text: 'Logs' },
   ]),
-  withUserAuthorizationPrompt([{action: 'cluster:status', resource: '*:*:*'}, {action: 'cluster:read', resource: 'node:id:*'}])
-)
-(class WzLogs extends Component {
-  constructor(props) {
-    super(props);
-    this.offset = 350;
-    this.state = {
-      isCluster: false,
-      selectedDaemon: '',
-      logLevelSelect: '',
-      descendingSort: false,
-      searchBarValue: '',
-      appliedSearch: '',
-      logsList: '',
-      isLoading: false,
-      offset: 0,
-      selectedNode: '',
-      logsPath: '',
-      nodeList: [],
-      totalItems: 0,
-      daemonsList: [],
-      loadingLogs: false,
-      realTime: false
+  withUserAuthorizationPrompt([
+    { action: 'cluster:status', resource: '*:*:*' },
+    { action: 'cluster:read', resource: 'node:id:*' },
+  ])
+)(
+  class WzLogs extends Component {
+    constructor(props) {
+      super(props);
+      this.offset = 350;
+      this.state = {
+        isCluster: false,
+        selectedDaemon: '',
+        logLevelSelect: '',
+        descendingSort: false,
+        searchBarValue: '',
+        appliedSearch: '',
+        logsList: '',
+        isLoading: false,
+        offset: 0,
+        selectedNode: '',
+        logsPath: '',
+        nodeList: [],
+        totalItems: 0,
+        daemonsList: [],
+        loadingLogs: false,
+        realTime: false,
+      };
+      this.ITEM_STYLE = { width: '300px' };
+      this.logger = new ErrorOrchestratorService();
```

Comment:
I suggest using 'errorHandler' or 'errorOrchestraror' instead of the logger to be more descriptive and asertive